packages/hooks/src/useIsMounted/useIsMounted.ts

```diff
@@ -0,0 +1,30 @@
+import { useLayoutEffect, useRef } from "react";
+
+/**
+ * Why does this work?
```

Comment:
Should this go in an MDX file? @andrewjdmccann

Reply:
It should but I think it should also be in this file.